### PR TITLE
Fixes #2381 Update AWS Cloud9 documentation

### LINKED ISSUE #2381: Outdated AWS Cloud9 documentation
[aws-cloud9](https://github.com/Seneca-CDOT/telescope/blob/master/docs/aws-cloud9.md) is outdated and one can simply not get Telescope running with this doc! Changes must be made!

### PR DESCRIPTION
**Pull request by cindyledev**
## Issue This PR Addresses
Closes #2381 Outdated AWS Cloud9 documentation

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR adds new instructions on how to run Telescope in development in the AWS Cloud9 IDE. This has successfully worked on t2.medium and t2.large EC2 instances.

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

<a href="https://gitpod.io/#https://github.com/Seneca-CDOT/telescope/pull/2390"></a>

### CODE REVIEW

File: tools/aws-ip.sh [hunk 1]
```diff
@@ -1,3 +1,8 @@
 ipv4=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
 
-echo '\n\nAPI_URL=http://'$ipv4':3000' >> ../telescope/.env
+cp config/env.development config/env.remote
+
+sed -i "s/for Development/for Remote Development/g" config/env.remote
+sed -i "s/localhost/$ipv4/g" config/env.remote
+
+cp config/env.remote .env
```
Comment (humphd) [inline tools/aws-ip.sh:8]: Nice fix, way simpler!  Let's `cp` the file once to `.env` and run `sed` on it there.

File: tools/aws-ip.sh [hunk 2]
```diff
@@ -1,3 +1,8 @@
 ipv4=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
 
-echo '\n\nAPI_URL=http://'$ipv4':3000' >> ../telescope/.env
+cp config/env.development config/env.remote
+
+sed -i "s/for Development/for Remote Development/g" config/env.remote
+sed -i "s/localhost/$ipv4/g" config/env.remote
```
Comment (humphd) [inline tools/aws-ip.sh:6]: I wonder if we should look for `http://localhost` vs. `localhost` on its own?  I'm trying to think if we'll ever regret changing `localhost` to an ip automatically, and breaking something in the future.  Maybe it's fine, I'm just pondering...

**Review by humphd — CHANGES_REQUESTED** [review]

**Comment by cindyledev** [comment]
This is ready for some daring reviewers. I used what I liked from @DukeManh 's Gitpod stuff here

Honourable mention #2439

**Comment by DukeManh** [comment]
I just tried AWS for the first time. It works!

**Comment by humphd** [comment]
@DukeManh care to leave a review for this?

**Review by DukeManh — APPROVED** [review]
Very detailed documentation.

File: docs/aws-cloud9.md [hunk 3]
```diff
@@ -236,6 +192,30 @@ Filesystem      Size  Used Avail Use% Mounted on
...
 ```
 
+## Install Docker-Compose
+
+By default, Docker is installed on AWS EC2's Ubuntu but Docker-Compose is not, so we have to install it ourselves.
+
+1. Run to download the current stable version of Docker-Compose:
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```
Comment (HyperTHD) [inline docs/aws-cloud9.md:202]: I'm gonna assume that the AWS instance will always be ran with linux in mind and never windows correct?

Comment (humphd) [reply docs/aws-cloud9.md:202]: I think that's a safe assumption, yes.

Comment (cindyledev) [reply docs/aws-cloud9.md:202]: For this particular AWS Cloud9 environment, we are using Ubuntu Server 18.04 LTS. When working with AWS EC2, it's common to use the Ubuntu, and Amazon Linux 2 operating systems. As of last year, you can actually use Amazon EC2 Mac Instances to build/test macOS, iOS, iPadOS, tvOS, and watchOS Apps. It was a major breakthrough.

**Comment by cindyledev** [comment]
These changes were included in #2398

**Pull request closed without merging**